_script/cibuild

```diff
@@ -13,6 +13,7 @@ if [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ "$TRAVIS_BRANCH" = "master" ]
 then
     echo "INFO: Merging to master... time to build and deploy redistributables"
     docker_name="dachad/tcpgoon"
+    export COMMIT_HASH=$(git rev-parse --short HEAD)
```

Comment:
you are in the right direction! But rather than exporting vars, just use arguments in the scripts